<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="datesTable">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>771</width>
      <height>481</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Next Day</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Previous Day</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Is Leap</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Week Number</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Duration</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="BDLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select BD</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="calendarBD">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>110</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="addLbl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Date To Add</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="calendarAdd">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>110</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="changeLbl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Date To Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Change </string>
    </property>
   </widget>
   <widget class="QDateEdit" name="calendarChange">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>110</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="birthdayDaysCounterLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>989</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
